File: c++/iniciante/src/2791.cpp
/* title: Feijão
* number of the problem: 2791
* date: 01/09/21
* author: ltsdw */

#include <iostream>


int main(int argc, const char* argv[])
{
    using std::cin, std::cout;

    for (int i = 0; i < 4; ++i)
    {
        int a;
        cin >> a;

        if (a)
        {
            cout << ++i << '\n';
            break;
        }
    }

    return EXIT_SUCCESS;
}

File: haskell/iniciante/2791.hs
-- title: Feijão
-- number of the problem: 2791
-- date: 01/09/21
-- author: ltsdw

resolve :: [Int] -> Int -> Int
resolve (x:xs) idx
    | x == 1 = idx
    | otherwise = resolve xs (idx+1)

main :: IO ()
main = do
    l1 <- getLine

    let xs = map (read :: String -> Int) $ words l1

    print $ resolve xs 1
